src/elements/form/extras/address/Address.js

```diff
@@ -73,17 +82,25 @@ export class NovoAddressElement implements ControlValueAccessor {
 
     updateControl() {
         this.onModelChange(this.model);
+        this.changeDetectorRef.markForCheck();
     }
 
     writeValue(model:any):void {
-        if (!this.model || !this.model.length) {
-            this.model = {
-                countryID: 1,
-                countryName: 'United States'
-            };
-        } else {
-            this.model = model;
-        }
+        setTimeout(() => {
```

Comment:
Without the `setTimeout` we get property has changed after it was checked errors. Should we add a check to only set `countryName` if it doesn't exist?